lib/app/file.js

```diff
@@ -1,10 +1,15 @@
 'use strict';
 
-var ArrayStore = require('fh-wfm-mediator/lib/array-store');
-
-var files = [];
+var SimpleStore = require('fh-wfm-simple-store');
 
 module.exports = function(mediator) {
-  var arrayStore = new ArrayStore('file', files);
-  arrayStore.listen('cloud:', mediator);
+  var fileStore = new SimpleStore('file');
+
+  fileStore.list().then(function(res) {
+    if (!res || res.length === 0) {
```

Comment:
I don't think you need initial `list` calls here. The fh.db implementation only touches the database upon `init(data)`, the constructor only initializes some local variables.